virt/nodes/virt-node-maintenance.adoc

```diff
@@ -82,14 +82,7 @@ endif::openshift-rosa,openshift-dedicated[]
 [id="run-strategies"]
 == Run strategies
 
-A virtual machine (VM) configured with `spec.running: true` is immediately restarted. The `spec.runStrategy` key provides greater flexibility for determining how a VM behaves under certain conditions.
-
-[IMPORTANT]
-====
-The `spec.runStrategy` and `spec.running` keys are mutually exclusive. Only one of them can be used.
-
-A VM configuration with both keys is invalid.
-====
```

Comment:
Instead of removing it, I would mention that using `spec.running` is not recommended since its deprecated in favor of using `runStrategy`

Reply:
same